doc/source/whatsnew/v0.21.1.txt

```diff
@@ -56,6 +56,8 @@ Documentation Changes
 
 Bug Fixes
 ~~~~~~~~~
+- Bug in :meth:`Series.fillna` which was raising RuntimeError when got large integer (:issue:`18159`).
```

Comment:
move this to Conversions section.

``Bug in :meth:`Series.fillna` which raised when passed a long integer on Python 2``